index.yaml

```diff
@@ -30,9 +30,26 @@ entries:
     urls:
     - https://orchestracities.github.io/charts/orion-0.1.0.tgz
     version: 0.1.0
+  pgadmin-chart:
+  - apiVersion: v1
+    created: 2019-04-17T16:53:21.919607+02:00
+    description: pgAdmin is the most popular and feature rich Open Source administration
+      and development platform for PostgreSQL, the most advanced Open Source database
+      in the world.
+    digest: c3f89e8ace5513098629ca00e8c203f4dbd12e9512ad9b013182d2834e3ea8dc
+    home: https://www.pgadmin.org/
+    keywords:
+    - pgadmin
+    - postgres
+    - database
+    - sql
+    name: pgadmin-chart
+    urls:
+    - https://orchestracities.github.io/charts/pgadmin-chart-0.1.0.tgz
```

Comment:
how do we generate this?

Reply:
i need to document it a bit, but basically "helm package <chartname>" then you issue another command to update the index. as said in a call, it can be all automated actually,

Reply:
I don't think there's a pressing need to automate that too, I was just curious :-)

Reply:
Agree, it'll be good to at least add those instructions in the docs